src/mndragossh/week3/task3/Task3Main.java

```diff
@@ -0,0 +1,23 @@
+package src.mndragossh.week3.task3;
+
+public class Task3Main {
+    public static void main(String[] args) {
+        User[] users = new User[]{
+                new User("First User", "first@gmail.com"),
+                new User("Second User", "second@gmail.com"),
+                new User("Third User", "third@gmail.com"),
+                new User("Fourth User", "fourth@gmail.com"),
+                new User("Fifth User", "fifth@gmail.com"),
+        };
+        Board board = new Board();
+        for (var user : users
+        ) {
+            board.postMessage(
+                    new Message(user,
+                            String.format("Hello I'm %s and this is %s message", user.getName(),
+                                    user.getName().split(" ")[0].toLowerCase())));

```

Comment:
e ok pentru valorile date de tine pentru numele userului, dar daca mai creeaza cineva obiecte si numele e null o sa iti apara un NullPointerException, sau daca nu foloseste un space in string, de aceea ar fi bine sa adaugi si niste validari in constructorul de la User, pentru ca daca lucrezi cu alti oameni la aceelasi proiect e posibil sa vina cineva si sa il foloseasca intr-un mod in care nu te asteptai si de-asta trebuie sa ai grija sa acoperi toate validarile